Makefile

```diff
@@ -15,7 +15,6 @@ GOTESTFLAGS?=-race
 GIT_SHA=$(shell git rev-parse HEAD)
 
 build: $(MICROSERVICES)
-	$(GOCGO) install -tags=safe
```

Comment:
Why was this line removed? It doesn't seem related to the snap build...

Reply:
This change is to be consistent with other device service.
Reverted the change to make this PR only snap-related.